proto/viam/app/data/v1/data.proto

```diff
@@ -226,6 +229,20 @@ message TabularDataByMQLResponse {
   reserved "data";
 }
 
+// GetLatestTabularDataRequest requests the most recent tabular data captured from the specified data source.
+message GetLatestTabularDataRequest {
+  string part_id = 1;
+  string resource_name = 2;
+  string method_name = 3;
+}
+
+// GetLatestTabularDataResponse provides the data, time synced, and time captured of most recent tabular data captured from the requested data source.
```

Comment:
Could you add a line in the comment here about how this will only return the latest data point if it was synced within the last year?